src/components/Collections/CreateCollections.svelte

```diff
@@ -1231,29 +1180,30 @@
       {/if}
     </div>
     <!-- Previous Memberships -->
-    <div class="grid grid-cols-3 justify-between gap-4">
+    <div
+      class="grid grid-cols-[repeat(auto-fit,_minmax(120px,_1fr))] justify-between gap-4"
+    >
       {#each collection.memberships as mem, i}
         {#if mem.id !== membership.id}
-        <div>
-          <MembershipOption
-          clubName={clubName ?? 'Your Club'}
-          id={mem.id}
-          name={mem.name}
-          imagePath={mem.imageSrc.trim().length > 0 ? mem.imageSrc : emptyDummyImage(400, 400)}
-          price={mem.price.toString()}
-          currency={mem.currency}
-          description={mem.description}
-          className={`lg:row-start-3 ${getColStart(i)}`}
-        />
-        <a
-        class="hs-button is-filled is-fullwidth mt-4 rounded px-8 py-6 text-base font-bold text-white"
-        href={`${collection.id}/${mem.id}`}
-      >
-        <span class="hs-button__label">Edit</span>
-      </a>
-        </div>
+          <div>
+            <MembershipOption
+              clubName={clubName ?? 'Your Club'}
+              id={mem.id}
+              name={mem.name}
+              imagePath={mem.imageSrc}
```

Comment:
```suggestion
             imagePath={mem.imageSrc.trim().length > 0 ? mem.imageSrc : emptyDummyImage(400, 400)}
```
helps accommodating empty image membership with emptyImage picture